jwala-services/src/main/java/com/cerner/jwala/service/media/impl/MediaServiceImpl.java

```diff
@@ -97,12 +105,13 @@ public void remove(final String name) {
 
     /**
      * This method will check for the existing jvm associations for the media
+     *
      * @param name media name
      */
     private void checkForJvmAssociation(String name) {
```

Comment:
Yes we are using this method to check the jvm association (please check line 106)